src/sec_certs/dataset/cc.py

```diff
@@ -479,7 +493,7 @@ def _get_primary_key_str(row: Tag):
                 x.st_link,
                 None,
                 None,
-                profiles.get(x.dgst, None),
+                None,
```

Comment:
Why the change? Does this mean we will have no links to PPs in the df?

Reply:
Hm, I will enrich the DF with one extra attribute for that!